packages/react-native/Libraries/Text/Text/RCTTextShadowView.mm

```diff
@@ -95,6 +95,17 @@ - (void)uiManagerWillPerformMounting
 
                          [descendantViewTags addObject:shadowView.reactTag];
                        }];
+  // [macOS
+  [textStorage enumerateAttribute:RCTTextAttributesTagAttributeName
```

Comment:
Can this more simply replace the enumeration done on lines 88-97?  Am I understanding the difference correctly that the first is looking for embedded Views inside the Text, ignoring all of the text runs?

Reply:
The only two options for enumerating attributes are [only one of them](https://developer.apple.com/documentation/foundation/nsattributedstring/1412461-enumerateattribute?language=objc) and [every single one](https://developer.apple.com/documentation/foundation/nsattributedstring/1412070-enumerateattributesinrange?language=objc). I wasn't aware of the latter until just now, and I can see how it would be nicer for more complicated `Text` elements.

Reply:
If the Set is necessary because we have known duplicates we will hit, what are they? Can we describe the problem so the next person understands the reasoning for these changes to be successful, e.g. if we were to seek pushing this change upstream?

Reply:
Realized a special case that will affect us with the workaround I'm investigating; the `<Text> <View> <Text/> </View> </Text>` pattern is going to create doubled events if the outer TextView generates enter/exit events for the inner true-leaf TextView (that's also generating events).

In such a situation, does the outer TextView return `IsYogaLeafNode() = true`?

Reply:
Yes, although it still returns true for the`RCTTextView` corresponding to the outermost gray box in the example mentioned in the test plan, so this may be a separate thing.

Reply:
Also, the case of `<Text> <View> <Text/> </View> </Text>` has been validated with the latest set of changes, so I think we can mark this as resolved.